boards/earlgrey-nexysvideo/.cargo/config

```diff
@@ -0,0 +1,2 @@
+[target.'cfg(target_arch = "riscv32")']
+runner = "qemu-system-riscv32 -M opentitan -bios ../../tools/qemu-runner/opentitan-boot-rom.elf -nographic -serial stdio -monitor none -semihosting -kernel "
```

Comment:
Does this need to be in `.cargo`? As a rule, we have avoided using hidden configuration in Tock (this is one of the underlying reasons for our make-based design). Can we drive these with flags from the invocation instead?

fwiw @jrvanwhy opened an upstream issue about this a little while ago: https://github.com/rust-lang/cargo/issues/8244, but it does not seem to have gained much traction :/

Reply:
Yeah, it does have to be. I couldn't get it to work anywhere else.